### DISCUSSION TITLE
Do you accept applications written in other programming languages?

### DISCUSSION THREAD

**Discussion by ghost** [elementary/appcenter#2054] [Q&A] [answered]
Hi! Do you accept publishing applications in GTK+Granite but written in other programming languages, such as Rust, as long as they look and behave exactly like a regular Vala application? Rust is more comfortable for me to code than Vala.

**Comment by davidmhewitt** [comment, marked as answer]
Yes, any language is fine as long as the GUI is Gtk3 or Gtk4.

See: https://docs.elementary.io/develop/appcenter/publishing-requirements#technical-requirements
